github/github_test.go

```diff
@@ -507,10 +507,16 @@ func TestNewRequest(t *testing.T) {
 		t.Errorf("NewRequest(%q) Body is %v, want %v", inBody, got, want)
 	}
 
+	userAgent := req.Header.Get("User-Agent")
+
 	// test that default user-agent is attached to the request
-	if got, want := req.Header.Get("User-Agent"), c.UserAgent; got != want {
+	if got, want := userAgent, c.UserAgent; got != want {
 		t.Errorf("NewRequest() User-Agent is %v, want %v", got, want)
 	}
+
+	if !strings.Contains(userAgent, c.PackageVersion) {
```

Comment:
Is there somewhere else I should add a test to help confirm that this is being added to _all_ requests? Or do you think this is sufficient?

Reply:
I think this is sufficient, thanks.